### ISSUE TITLE
Feature Request: Enable using institute access to get full text PDF

### ISSUE THREAD

**Issue by RamRS** [ContentMine/getpapers#39]
Not sure if this can be applied to all types of institute access. My current institute uses SSO, so that could be a bit easy, but form based SSO, such as the one at NYU (and many other institutes) might be a bit more challenging.

**Comment by Blahah** [comment]
@RamRS any idea how your SSO works? Does it give you a cookie for example?

**Comment by RamRS** [comment]
My current SSO? I'm not sure - from outside my office network, I log in with DOMAIN\user and password. Should I check if I have cookies from my office domain?

**Comment by RamRS** [comment]
So, I just tried accessing nature, and the URL went from `nature.com/<REST_OF_URL>` to `<INSTITUTE_PORTAL>:<PORT>/<REST_OF_URL>`

**Comment by Blahah** [comment]
OK thanks - that looks like it might be tractable. How do you log in - through a form in a web interface?

**Comment by RamRS** [comment]
Yes, I search for a journal on a public site. When I click on the search result, it redirects to the second URL pattern seen above. I think access to that server is controlled through my institute's AD.

You know what, this protocol works for all my previous schools as well. NYU did this exact same `<ROUTING_URL>:<PORT>/<REST_OF_URL>` thing, where the first part would substitute the journal website.

My grad school (a part of NYU now, the migration happened when I was in school) used to add a `.databases.poly.edu` to the actual journal's URL (E.g: `nature.databases.poly.edu`) and not display the routing URL or the port used.

I also think the port is a bit dynamic - as in, I'm not sure if port X mapped to journal Y all the time. Maybe they were re-mapped periodically, I'm not sure.

**Comment by Blahah** [comment]
OK thanks - can you give me an example link with the routing url/port intact? I assume they can't be used without logging in first, but I'd like to see what HTTP headers there are.

**Comment by RamRS** [comment]
Sure. Here we go:

Mount Sinai: 
Nature: http://eresources.library.mssm.edu:2155/nature/archive/index.html
Science: http://eresources.library.mssm.edu:2145/magazine

Tried changing port, leads to different journals.

My NYU access has been revoked.
